### PR TITLE
Job processing

### PR DESCRIPTION
**Pull request by ryanlabouve**
Not super happy with data flow ATM, but this does work.

Doing regular polling to update the job status to correspond with changes on the backend.

### CODE REVIEW

**Comment by ryanlabouve** [comment]
#24

File: app/components/building-job-status/component.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    return Ember.run.later(this, function() {
+      f.apply(this);
+      this.set('timer', this.schedule(f));
+    }, this.get('interval'));
+  },
+
+  didInsertElement() {
+    this.set('timer', this.schedule(this.get('onPoll')));
+    this.get('buildingJobStatus').registerPoll(() => {
+      debugger;
```
Comment (stevenwu) [inline app/components/building-job-status/component.js:16]: Forgot to remove debugger

Comment (ryanlabouve) [inline app/components/building-job-status/component.js:16]: Eff. I know I left one in there somewhere. 🙇

**Comment by johnmosesman** [comment]
Merging and I will remove the `debugger` on the main repo where I have push access

**Pull request merged**